src/tests/Favorites.test.js

```diff
@@ -8,9 +8,11 @@ describe('Favorites', () => {
       render(<Favorites />);
     });
 
-    // test('must show favorite videos', () => {
-    //   render(<Favorites />);
-    // });
+    test('must be able to retrieve from local storage', () => {
```

Comment:
I think this is not needed if you don't render your component your test will pass anyway.